css/selectors4/focus-within-001.html

```diff
@@ -6,7 +6,6 @@
 <link rel="author" title="Florian Rivoal" href="mailto:florian@rivoal.net">
 <link rel="help" href="https://drafts.csswg.org/selectors-4/#focus-within-pseudo">
 <link rel="match" href="focus-within-001-ref.html">
-<meta name="flags" content="interact">
```

Comment:
* It is better to switch to `<meta name="flags" content="">` than to remove the line. Makes it clear that you have considered the question and decided that there was no need for flags, instead of maybe forgetting to think about it.

* If you remove the interact flag, the comment above the javascript about it being an optional convenience becomes wrong and needs to be removed: either the test is interactive and the js is a optional helper, or it is automated and js is required.

* As we switch to js required, it would be good to add `class="reftest-wait"` on the root, and remove it once js has run, so that we make sure that the ref comparison is done at the right time, and that if js fails to run for some reason, we detect it as a test that failed to run rather than a failure to implement focus-within correctly.